docs/prepping-local-system-runbook-windows.md

```diff
@@ -31,17 +31,25 @@ Where \<platform\>-demo is:
 
 - EKS-demo - for the EKS runbook
 
-- GKE-demo - for the GKE runbook
-
 - PKS-demo - for the PKS runbook
 
 - Openshift-demo - for the Openshift runbook
 
+Currently there is no runbook for running on the Google Kubernetes Engine (GKE) using the Windows 10 Google SDK. To set up a system for a deployment on GKE, see [Prepare a local Linux system – 45 minutes](docs/prepping-local-system-runbook-linux.md). 
+
 You are ready to continue preparing your local system.
 
 Installing required applications for the deployment
 ---------------------------------------------------
 
+The entire deployment requires the following applications to be used a somee point during the process, so it's useful to prepare your local system with all of the files before you start your deployment:
+- Helm
+- kubectl
+- Docker
+- unzip (or something equivalent to extract files from .zip archive files.)
```

Comment:
Should we recommend 7zip?

Reply:
for windows, the unzip command is there by default. I didn't want to go specifically into adding a tool when Windows would do it...I think we're covered with (or something equivalent to extract files from .zip archive files.)